### PR TITLE
chore: adhere to clean architecture rules (no dto in domain)

### PR DESCRIPTION
**Pull request by kuamanet**
### TLDR;
I basically moved out dtos & mappers from the domain

### Why

From the [The Clean Code Blog](https://blog.cleancoder.com/uncle-bob/2012/08/13/the-clean-architecture.html)

> Interface Adapters
The software in this layer is a set of adapters that convert data from the format most convenient for the use cases and entities, to the format most convenient for some external agency such as the Database or the Web. It is this layer, for example, that will wholly contain the MVC architecture of a GUI. The Presenters, Views, and Controllers all belong in here. The models are likely just data structures that are passed from the controllers to the use cases, and then back from the use cases to the presenters and views.

> *Similarly, data is converted, in this layer, from the form most convenient for entities and use cases, into the form most convenient for whatever persistence framework is being used*

More over:

> What data crosses the boundaries.
... when we pass data across a boundary, it is always in the form that is most convenient for the inner circle.

### CODE REVIEW

File: web-api/src/Kerbero.Domain/Kerbero.Domain.csproj [hunk 1]
```diff
@@ -12,4 +12,8 @@
       <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="7.0.0" />
     </ItemGroup>
 
+    <ItemGroup>
+      <Folder Include="SmartLocks\Mappers" />
+    </ItemGroup>
```
Comment (vash72) [inline web-api/src/Kerbero.Domain/Kerbero.Domain.csproj:17]:
```suggestion

```

**Review by vash72 — CHANGES_REQUESTED** [review]
need to be rebased

**Review by vash72 — APPROVED** [review]
